### PR TITLE
Add Dockerfile for developers to use to run Chasten

### PR DESCRIPTION
**Pull request by AidanNeeson**
Includes the addition of a Dockerfile that allows for developers to run Chasten without the fuss of OS specific dependency issues. Issue #40 includes the steps necessary to create the container, run, and manage it. A wiki would be helpful to document these steps so they will not clog up the README.

### CODE REVIEW

**Review by jnormile — APPROVED** [review]
Super useful for getting remaining members of the dev team up and running! Thanks for putting this together!

Alish is looking at integrating the information from the issue tracker about building/running the container into the chasten documentation that will be built by Quarto and hosted via GitHub Pages.

**Review by gkapfham — CHANGES_REQUESTED** [review]
Can you please include some documentation, perhaps in the README.md file, about how to use this Dockerfile? Without the specific command for using Docker with this specific Docker container, I'm concerned that this will not see significant use. Thanks! Finally, @AidanNeeson you are welcome to assign this issue to someone in our team.

**Comment by jnormile** [comment]
The added content looks good to me--appreciate the different commands for multiple OSs being explicitly stated!

**Review by laurennevill — APPROVED** [review]
I believe this could be really helpful for those who are still struggling to get the project working on their machines. Great work

**Review by Finley8 — APPROVED** [review]
From personal experience of using DockerFile on my computer, this fully works and is very helpful.

**Comment by gkapfham** [comment]
> From personal experience of using DockerFile on my computer, this fully works and is very helpful.

Hi @Finley8, thanks for confirming. Looking forward to future PR   reviews that you may complete, I suggest that you also furnish evidence that the command(s) worked by showing program output in your terminal window in the PR review. See what I mean? Overall, the more concrete output that we can provide the better we can establish a confidence in the correctness of the PR.

File: README.md [hunk 1]
```diff
@@ -89,6 +89,25 @@ Follow these steps to install the `chasten` program:
 - Type `pipx list` and confirm that Chasten is installed
 - Type `chasten --help` to learn how to use the tool
 
+## 🐋 Docker
+
+There is also the option to use [Docker](https://www.docker.com/) to interact with `chasten`
+
+Follow these steps to utilize Docker:
+
+- Install Docker Desktop for your operating system
```
Comment (gkapfham) [inline README.md:98]: Should we provide a link to Docker Desktop? Maybe this link?

https://docs.docker.com/get-docker/

File: README.md [hunk 2]
```diff
@@ -89,6 +89,25 @@ Follow these steps to install the `chasten` program:
 - Type `pipx list` and confirm that Chasten is installed
 - Type `chasten --help` to learn how to use the tool
 
+## 🐋 Docker
+
+There is also the option to use [Docker](https://www.docker.com/) to interact with `chasten`
```
Comment (gkapfham) [inline README.md:94]: Instead of the words "interact with", I suggest that we write "use". See what I mean?

**Review by gkapfham — CHANGES_REQUESTED** [review]
Hi @AidanNeeson, thanks for preparing this `Dockerfile`!

I have suggested two small phrasing improvements. Can someone on the team make these small changes?

Also, we don't need to make this change now: but, maybe later at the top of the README file we should mention Docker?

**Review by gkapfham — APPROVED** [review]
Approved, thanks for making the requested changes.

**Review by laurennevill — APPROVED** [review]

**Review by hayleepierce — APPROVED** [review]

**Pull request merged**